app/views/jobseekers/profiles/show.html.slim

```diff
@@ -6,6 +6,12 @@
       = t(".page_title")
       = govuk_tag(text: t(".active"), colour: "green", classes: "vertical-align-middle govuk-!-margin-left-1") if profile.active?
 
+    - unless current_jobseeker.account_merge_confirmation_code
```

Comment:
To confirm I understood this right.
This condition will show the banner to all the One Login users except if:
-  the user created a second account (B)
- requested this account (A) to be merged into B
- still signed in with account A before executing the account transfer A -> B.